thirdparty/lua-htmlparser/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+project(lua-htmlparser)
+cmake_minimum_required(VERSION 3.5.1)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/../cmake_modules")
+include("koreader_thirdparty_common")
+include("koreader_thirdparty_git")
+
+enable_language(C)
+
+assert_var_defined(CC)
+assert_var_defined(CFLAGS)
+assert_var_defined(LD)
+assert_var_defined(LUA_INCDIR)
+assert_var_defined(LUA_LIBDIR)
+assert_var_defined(LUA_HTMLPARSER_VER)
+assert_var_defined(OUTPUT_DIR)
+assert_var_defined(LUAROCKS)
+
+ep_get_source_dir(SOURCE_DIR)
+
+set(LUA_HTMLPARSER_ROCKSPEC rockspec/${LUA_HTMLPARSER_VER}.rockspec)
```

Comment:
Ought to be `rockspec/htmlparser-${LUA_HTMLPARSER_VER}.rockspec` instead, right? (c.f., the failing gcc-debug bot?)

Reply:
Indeed.